utils/_context/_scenarios.py

```diff
@@ -245,6 +244,45 @@ def configure(self, replay):
         for container in reversed(self._required_containers):
             container.configure(replay)
 
+    def _start_interface_watchdog(self, watches=[]):
+        if not watches:
+            return
+
+        from threading import Thread
+
+        # Lots of issue using the default OS dependant notifiers (not working on WSL, reaching some inotify watcher limits on Linux). Using a very simple poller instead.
+
+        def create_watchdog(parent):
```

Comment:
No need to. This case is handled as before: `ingest_file` will fail, and it'll get a chance to do it on next iteration. It is guaranteed to be retried as long as the watchdog is running, since it simply calls `ingest_files` for all files ending in `.tmp`, whether they are new or not.